src/components/Link/Link.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+import * as PropTypes from 'prop-types';
+import * as React from 'react';
+
+export default class Link extends React.Component {
+  render() {
+    const help = this.props.helpIcon
+      ? (
+        <chi-icon
+          icon="circle-question-outline"
+          onClick={this.props.clickHelp}
+          onMouseEnter={this.props.mouseoverHelp}
+          onMouseLeave={this.props.mouseleaveHelp}>
+        </chi-icon>
+      )
+      : <chi-icon></chi-icon>;
```

Comment:
Let's import the native react <Icon /> component and reuse it here to solve the empty space issue.
Here is an example

https://user-images.githubusercontent.com/25136131/124928190-b5855200-dfff-11eb-855a-7f811738aad0.mov